hack/tools/go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/golangci/golangci-lint v1.21.0
 	github.com/jteeuwen/go-bindata v3.0.7+incompatible
+	golang.org/x/exp v0.0.0-20190312203227-4b39c73a6495 // indirect
```

Comment:
Do you know where this is coming from? cc @vincepri

Reply:
Looks like you need to run `make modules`

Reply:
I'm not totally sure, we shouldn't be using anything from the experimental go package afaik, unless we mistakenly updated some deps

Reply:
Ran `make verify-modules` and it removed the redundant module. Pushed the change, waiting for the build to go through.